### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -66,7 +66,7 @@ etcd_build() {
       "-ldflags=${GO_LDFLAGS[*]}" \
       -o="../${out}/etcdctl" . || return 2
   ) || return 2
-  # Verify whether symbol we overriden exists
+  # Verify whether symbol we overridden exists
```
Comment (ahrtr) [inline scripts/build.sh:69]: overwrote

Comment (cuishuang) [reply scripts/build.sh]: Thanks ^_^

Done

**Comment by ahrtr** [comment]
Thank you. Please squash the two commits into one.

**Comment by cuishuang** [comment]
> Thank you. Please squash the two commits into one.

Done

**Review by ahrtr — APPROVED** [review]
LGTM

**Pull request merged**